backend/audit/test_models.py

```diff
@@ -103,6 +110,25 @@ def test_submission_status_transitions(self):
 
                         self.assertRaises(TransitionNotAllowed, transition_method)
 
+    def test_no_late_changes(self):
+        """
+        Try to make a change to a submission with a status that isn't
+        in_progress and get an expected error.
+        """
+        bad_statuses = [
+            status[0]
+            for status in SingleAuditChecklist.STATUS_CHOICES
+            if status[0] != SingleAuditChecklist.STATUS.IN_PROGRESS
+        ]
+
+        for status_from in bad_statuses:
+            sac = baker.make(
+                SingleAuditChecklist, audit_type="annual", submission_status=status_from
```

Comment:
I think we're mixing up audit type and audit period here

Reply:
Yup, I changed that to actually have audit types here.